src/parse_data.jl

```diff
@@ -125,4 +125,135 @@ function load_case(name, agg_nodes, B, nodes; resources=BASE_RESOURCES)
     end
     
     return d, g, case
+end
+
+"""
+    load_synthetic_network(case_name)
+"""
+function load_synthetic_network(case_name)
+    # Load network data
+    network_data = parse_file(joinpath("../data", case_name));
+	net = make_basic_network(network_data)
+	make_per_unit!(net)
```

Comment:
I like this indentation style. In Julia, what does it mean? Is it like a paragraph?

Reply:
Ah, just a typo :)